pkg/crc/config/viper_config.go

```diff
@@ -85,13 +83,6 @@ func (c *ViperStorage) BindFlagSet(flagSet *pflag.FlagSet) error {
 	return c.viper.BindPFlags(flagSet)
 }
 
-func (c *ViperStorage) refresh() error {
```

Comment:
This was added in the previous commits, maybe the commit adding this can be dropped?

Reply:
Yes, I kept it to show where I come from. I will squash this when it will be ready to merge.

Reply:
I feel like a broken record, but mentioning that the `refresh()` is only temporary and should be squashed before merging is really useful info to know at review time :)